cms/core/models/snippets.py

```diff
@@ -11,7 +12,7 @@ class ContactDetails(index.Indexed, models.Model):
     Note that this is registered as a snippet in core.wagtail_hooks to allow customising the icon.
     """
 
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
We don't have the requirements for this. I can see a case where they may have the same name but different email addresses. Given that both the name and email are mandatory, I suggest we use a unique constraint based on these two values. Thoughts?

Reply:
still trying to get confirmation. will make name+email unique together

Reply:
That makes sense to, but then it would be good to add the email in another column in the table on the page so they can be easily distinguished.

Reply:
72f51bd now adds:

![Screenshot 2024-12-10 at 11 43 09](https://github.com/user-attachments/assets/afb347da-ee01-4290-a465-0a0e17df6168)
![Screenshot 2024-12-10 at 11 43 16](https://github.com/user-attachments/assets/330ec840-7c42-455e-ab5b-f3d9703f73e4)

Reply:
Haven't run it myself yet, but on the screenshot the name column header seems to be inconsistent between the two pages - probably should be "Name" for both.

Reply:
Good spot @kacperpONS, fixed in 8d9f466